client/my-sites/plugins/plugin.jsx

```diff
@@ -318,18 +319,8 @@ const SinglePlugin = React.createClass( {
 	render() {
 		const { selectedSite } = this.props;
 
-		if ( this.state.accessError ) {
-			return (
-				<MainComponent>
-					{ this.renderDocumentHead() }
-					<SidebarNavigation />
-					<EmptyContent { ...this.state.accessError } />
-					{ this.state.accessError.featureExample
-						? <FeatureExample>{ this.state.accessError.featureExample }</FeatureExample>
-						: null
-					}
-				</MainComponent>
-			);
+		if ( ! this.props.userCanManagePlugins ) {
+			return <NotAvailableError />;
```

Comment:
This change introduces a regression - if you try to load `/plugin/:plugin/:site` with a clean Redux state, you'll see the following:

![](https://cldup.com/uUKXlLKwQo.png)

While in production, you can see the plugin information while the sites are loading.